public/content/eips/index.md

```diff
@@ -54,10 +54,20 @@ Full standardization process alongside with chart is described in [EIP-1](https:
 
 If you’re interested to read more about EIPs, check out the [EIPs website](https://eips.ethereum.org/) and [EIP-1](https://eips.ethereum.org/EIPS/eip-1). Here are some useful links:
 
+### Official EIP resources
+
 - [A list of every EIP](https://eips.ethereum.org/all)
 - [A description of all EIP types](https://eips.ethereum.org/EIPS/eip-1#eip-types)
```

Comment:
It is all listed on the [page](https://eips.ethereum.org). I don't see a need for listing them separately. 
Instead, we can add info on related meetings.

Reply:
> Instead, we can add info on related meetings.

What meetings do you have in mind--the [EIPIP meetings](https://github.com/ethcatherders/EIPIP)? I suppose we could add that in somewhere depending on how discussions over the structure for the listing play out.